bitnami/mongodb/templates/hidden/statefulset.yaml

```
@@ -494,6 +506,14 @@ spec:
             - key: mongodb-ca-key
               path: mongodb-ca-key
               mode: 0600
+        {{- else }}
+        {{- range $index, $secret := .Values.tls.hidden.existingSecrets }}
+        - name: mongodb-certs-{{ $index }}
+          secret:
+            secretName: {{ tpl $secret $ }}
```

Our charts usually make use of our own `common` function for rendering templates, could you please use it here 😁 ?

```suggestion
            secretName: {{ include "common.tplvalues.render" ( dict "value" $secret "context" $) }}
```

---

Sure :)